src/app/common/tabs.component.ts

```diff
@@ -286,6 +301,34 @@ export class TabsComponent implements OnDestroy, AfterViewInit {
         this._menuOn = (show == null) ? true : show;
     }
 
+    private toggleMenu() {
+        if (this._menuOn) {
+            this.showMenu(false);
+            return;
+        }
+        this.showMenu(true);
+    }
+
+    private onClickToggleMenu() {
+        this._toggleOnProcessing = true;
+
+        window.setTimeout(() => {
```

Comment:
to prevent double trigger, try to pass the event in the click and call `e.preventDefault()`

Reply:
And also check for `e.defaultPrevented` on toggle menu

Reply:
preventDefault() does not work for this case. I will open a new issue to investigate the issue that both click and enter key event happens even though I clicked only enter key.